var/spack/repos/builtin/packages/fides/package.py

```diff
@@ -0,0 +1,42 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Fides(CMakePackage) :
+    """A library that provides a schema for ADIOS2 streams.
+    """
+    homepage = "https://gitlab.kitware.com/vtk/fides"
+    url      = "https://gitlab.kitware.com/vtk/fides/-/archive/v1.0.0/fides-v1.0.0.tar.gz"
+    git      = "https://gitlab.kitware.com/vtk/fides.git"
+
+    maintainers = ['caitlin.ross', 'dpugmire']
```

Comment:
The first maintainer doesn't seem to be a valid Github username
```suggestion
    maintainers = ['dpugmire']
```